### DISCUSSION TITLE
why the location=2 time index_ini and patch_size?

### DISCUSSION THREAD

**Discussion by Fuzhiyuan** [TorchIO-project/torchio#805] [General]
i am reading torchio source code, and I do not understand why the location = index_ini*2+patch_size? in Sampler.py line:54.

**Comment by fepegar** [comment]
Hi, @Fuzhiyuan. Can you share a link to that file? I can't find that line you mentioned.

**Reply by Fuzhiyuan** [reply]
https://github.com/fepegar/torchio/blob/main/torchio/data/sampler/sampler.py   
the origin code is not index_ini*2, i just combine it.  I'm sorry if that's what kept you from finding it

**Comment by fepegar** [comment]
`location` is a list containing the indices of the first and last voxels in
the patch.
